### PR TITLE
Make weight decay configurable and change default value

### PR DESCRIPTION
**Pull request by SamitHuang**
- Previous bug:  
     weight decay is fixed to **1e-6**  
     The reference value used in LAION training or LoRA fine-tune in diffusers is 1e-2 

- Fix in this PR:  weight decay is now configurable via arg parser.
         Now default weight decay is  **1e-2**

### CODE REVIEW

File: examples/stable_diffusion_v2/scripts/run_train_v2.sh [hunk 1]
```diff
@@ -9,9 +9,9 @@ device_id=0
 
 output_path=output/finetune_pokemon
 task_name=txt2img
-data_path=./datasets/diffusion/pokemon_blip/train
+data_path=./datasets/pokemon_blip/train
 #data_path=/home/yx/datasets/diffusion/pokemon
-epochs=20
+epochs=10
```
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/scripts/run_train_v2.sh:14]: is this related too?

File: examples/stable_diffusion_v2/scripts/run_train_v2.sh [hunk 2]
```diff
@@ -47,4 +49,5 @@ python train_text_to_image.py \
     --epochs $epochs \
     --warmup_steps $warmup_steps \
     --image_filter_size=$image_filter_size \
+    --weight_decay=$weight_decay \
```
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/scripts/run_train_v2.sh:52]: config files or args?

**Review by DongyaoZhu — APPROVED** [review]
please see comments
